resources/lib/svtplay.py

```diff
@@ -220,7 +221,7 @@ def start_video(self, video_url):
         self.__resolve_and_play_video(video_json)
 
     def __add_directory_item(self, title, params, thumbnail="", folder=True, live=False, info=None, fanart=""):
-        list_item = xbmcgui.ListItem(title)
+        list_item = xbmcgui.ListItem(strip_html_tags(title))
```

Comment:
Sorry for not mentioning earlier, just realized this. 

Wouldn't it be better to strip the title in `GraphQL` instead of here, as is done with plot/description?

That way the "internal API" (GraphQL) is safe from HTML tags. I think that would be an improvement.